istioctl/cmd/istioctl/convert/testdata/v1alpha3/destination-rule-helloworld.yaml.golden

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  creationTimestamp: null
+  name: helloworld-service
+  namespace: default
+spec:
+  host: helloworld-service
+  trafficPolicy:
```

Comment:
Ah, I don't think `convert.DestinationPolicies` is converting correctly. It is not taking into account the `destination` of the `DestinationPolicy`. There should be a `version: v1` subset with the traffic policy.